### DISCUSSION TITLE
How can a Tableview double-click to get the content of a row?

### DISCUSSION THREAD

**Discussion by zhenzi0322** [israel-dryer/ttkbootstrap#347] [Q&A]
How can a Tableview double-click to get the content of a row?

**Comment by ghost** [comment]
I don't know if a solution was ever found for this but just in case it helps someone else you can do this:

When creating the table, add the binding along the lines of:

`self.my_tableview.view.bind("<Double-1>", self.row_selected) `

With a method of:

```
def row_selected(self, event: Any) -> None:
        self.selected_rows: list[Any] = self.my_tableview.get_rows(selected=True)
```
(I am not sure of the datatype of the selected rows - Any will do for now)

This will give a list of selected rows.

**Comment by marian-platonov** [comment]
Try with:

```
def on_double_click(event):
    headers = [col.headertext for col in table.tablecolumns]
    selected = table.view.selection()
    records = []
    for iid in selected:
        record: TableRow = table.iidmap.get(iid)
        records.append(record.values)
    print(records)
    

table.view.bind('<Double-1>', on_double_click)
```
